### ISSUE TITLE
Windows Device Recovery Tool invalid

### ISSUE THREAD

**Issue by johnwildes** [MicrosoftDocs/Hololens#19]
It doesn't appear that the Windows Device Recovery Toolkit is even available anymore.  This section of the document should be removed.  I have a gen 1 hololens, and need to update it.  In the past I could use the WDRT app to do this, now it won't connect to services online, and you cannot download newer versions of it.  

Is there another way to get this tool specifically for HoloLens gen1 replacement?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c6c41345-fc21-811f-b32f-b0f894e5a29d
* Version Independent ID: 8216737b-bb73-f5fa-8d1e-100e042bf72e
* Content: [Restart, reset, or recover HoloLens 1](https://docs.microsoft.com/en-us/hololens/hololens1-recovery)
* Content Source: [hololens/hololens1-recovery.md](https://github.com/MicrosoftDocs/Hololens/blob/public/hololens/hololens1-recovery.md)
* Product: **hololens**
* Technology: **windows**
* GitHub Login: @evmill
* Microsoft Alias: **v-evmill**

**Comment by evmill** [comment]
Hello John. Thanks for pointing this out. I'm reaching out to people to find out the owner for this link and get it working again.

**Comment by evmill** [comment]
@johnwildes The link has been fixed and should be working as expected now. Can you confirm for me that you are unblocked?
Thanks,
-Evan

**Comment by evmill** [comment]
Closing since this issue is resolved. Please feel free to comment if this issue is still occurring for you or re-open a new issue.

**Comment by johnwildes** [comment]
@evmill thanks for updating this, just checked the link, and was able to download the tool.
